.github/workflows/cd-sql-engine-populate-persistent-source-schema.yaml

```diff
@@ -0,0 +1,97 @@
+# See [Persistent Source Schema](/GLOSSARY.md#persistent-source-schema)
+# Populating the source schema via this workflow ensures that it's done with the same settings as the tests.
+
+name: Reload Test Data in SQL Engines
+
+# We don't want multiple workflows trying to create the same table.
+concurrency:
+  group: POPULATE_PERSISTENT_SOURCE_SCHEMA
+  cancel-in-progress: true
+
+on:
+  push:
+    branches:
+      - main
```

Comment:
Do we really want this to run on every push to main? I feel like that's asking for concurrency problems where one job set is dropping and re-creating persistent source schemas while another job set is trying to read from that schema. The dominant case is going to be the one where a push to main triggers no changes, so doing all of that work in the db feels like it'd be both wasteful and contributing to flaky engine test results.

Given how rare updates to the persistent source schemas are likely to be, I think we'd be better off with this triggered on `label` and `workflow_init` or whatever the workflow page setting is so we can manually kick this off on a per-branch basis via the GitHub UI. That way if main gets busted it's easy to just update the schema without labeling a PR, and for any PR making changes it's easy to update the schema with a label there.

Reply:
To put this another way - in theory our dominant use case is to run on push to main. If we settle back into that routine, what's the point of a persistent source schema if we repopulate it every time we push to main?

Reply:
Yeah, the intended behavior was for it to be a manually triggered operation. I got this section from the SQL engine tests workflow, which I thought had the same behavior, but apparently not.

Reply:
Also seems like this would not have taken action generally since we don't push to `main`.

Reply:
NVM push happens on merge.